stories/atoms/components/button/Button.stories.mdx

```diff
@@ -70,155 +67,204 @@ that will help users to find the action they're looking for.
-        <Button backgroundColor="error" variant="primary" label="Error" size="medium" />
-        <Button backgroundColor="warning" variant="primary" label="Warning" size="medium" />
-        <Button backgroundColor="info" variant="primary" label="Info" size="medium" />
-      </div>
-    )}
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        gap: '25px',
+        flexWrap: 'wrap'
+      }}
+    >
+      <Button backgroundColor="primary" variant="primary" label="Primary" size="medium" />
+      <Button backgroundColor="secondary" variant="primary" label="Secondary" size="medium" />
+      <Button backgroundColor="success" variant="primary" label="Success" size="medium" />
+      <Button backgroundColor="error" variant="primary" label="Error" size="medium" />
+      <Button backgroundColor="warning" variant="primary" label="Warning" size="medium" />
+      <Button backgroundColor="info" variant="primary" label="Info" size="medium" />
+    </div>
   </Story>
 </Canvas>
 
 ## Disabled
 
-The `Button` can be disabled at any time, thus preventing any user interaction.
+**Button** can be `disabled` at any time, thus preventing any user interaction.
 
 <Canvas>
   <Story name="Disabled">
-    {args => (
-      <div
-        style={{
-          display: 'flex',
-          alignItems: 'center',
-          justifyContent: 'center',
-          gap: '25px',
-          flexWrap: 'wrap'
-        }}
-      >
-        <Button disabled variant="primary" label="I'm disabled" size="medium" />
-      </div>
-    )}
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center'
+      }}
+    >
+      <Button disabled variant="primary" label="I'm disabled" size="medium" />
+    </div>
   </Story>
 </Canvas>
 
-## Content
+## Left / Right Icon
+
+`leftIcon` and `rightIcon` are auxilliary elements giving great freedom to embellish the button
+together with the `label`.
+
+<Canvas>
+  <Story name="Left / Right Icon">
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        flexWrap: 'wrap',
+        gap: '50px'
+      }}
+    >
+      <Button leftIcon={<Icon name="wallet" size={22} />} label="Get wallet address" />
+      <Button label="Add wallet" rightIcon={<Icon name="add" size={22} />} />
+    </div>
+  </Story>
+</Canvas>
 
-The content can be customized to fit users needs through its properties `label`, `icon`,
-`leftIcon` and `rightIcon`.
+## OnClick
 
-- `label` is the only obligatory property and is used to clearly indicate what action will be triggered, through the
-  tooltip and text.
-- `icon` sets the icon for the **variant** 'icon'.
-- `leftIcon` and `rightIcon` are auxilliary elements giving great freedom to embellish the button
-  together with the label.
+`onClick` is what makes the magic happen, without it the button has no corresponding action.
+The function passed as value should correspond to the action described in `label`.
```

Comment:
I do not understand this sentence..
What is the link between the callback function and the action described in `label`?

Reply:
I have changed the description and removed the mention of `label`.